Project.toml

```
@@ -1,7 +1,7 @@
 name = "NetworkProtocols"
 uuid = "cc8e9863-198f-4c11-8208-8a22403099d5"
 authors = ["Christian Rorvik <christian.rorvik@gmail.com>"]
-version = "0.2.5"
+version = "0.3.5"
```

We haven't done a 1.0 release here yet, so bumping the first non-zero is for breaking changes, patch/feature changes are the same (last component)
```suggestion
version = "0.2.6"
```